components/HeaderBackground/index.js

```diff
@@ -15,6 +15,7 @@ const StyledCircle = styled.div`
   height: 300px;
   border-radius: 50%;
   background-color: var(--highlight-red);
+  color: var(--white);
```

Comment:
I'd probably argue that `--white` is not really a variable (or should be one). If it's just `#fff` then perhaps describing what that white is used for would be more apt.